### ISSUE TITLE
Model's accuracy is different with your report on README.md when I evaluate the models.

### ISSUE THREAD

**Issue by dbwp031** [chenyaofo/pytorch-cifar-models#13]
Hi, I tried to evaluate your models' accuracy in two ways. In your report, `cifar100_MobileNetV2_x1_0` model accuracy is 74.20, but when I tried
1. evaluate by your [`start_on_colab`](https://github.com/chenyaofo/pytorch-cifar-models/blob/master/colab/start_on_colab.ipynb),
2. and evaluate by my own code

```
import torch
from torchvision import datasets, transforms
# data will be downloaded in data_directory.
data_directory = './data'
batchsize=256
device = 'cuda'
normalize = transforms.Normalize(mean=[0.507, 0.4865, 0.4409],
                                 std=[0.2673, 0.2564, 0.2761])

train_dataset = datasets.CIFAR100(root=data_directory,
                                  train=True,
                                  transform=transforms.Compose([
                                                                  transforms.RandomCrop(32, padding=4),
                                                                  transforms.RandomHorizontalFlip(),
                                                                  transforms.ToTensor(),
                                                                  normalize]),
                                  download=True)
train_loader = torch.utils.data.DataLoader(dataset=train_dataset,
                                           batch_size=batchsize,
                                           shuffle=True,
                                           pin_memory=True,
                                           num_workers=2)

test_dataset = datasets.CIFAR100(root=data_directory,
                                 train=False,
                                 transform=transforms.Compose([
                                                                  transforms.ToTensor(),
                                                                  normalize]),
                                 download=True)

test_loader = torch.utils.data.DataLoader(dataset=test_dataset,
                                          batch_size=batchsize,
                                          shuffle=False,
                                          pin_memory=True,
                                          num_workers=2)


def validate(model,loader):
    global device
    model.eval()
    correct = 0.
    total = 0.
    for images, labels in loader:
        images = images.to(device)
        labels = labels.to(device)

        with torch.no_grad():
            pred = model(images)

        pred = torch.max(pred.data, 1)[1]
        total += labels.size(0)
        correct += (pred == labels).sum().item()

    val_acc = (correct / total)*100
    model.train()
    return val_acc
model = torch.hub.load("chenyaofo/pytorch-cifar-models", "cifar100_mobilenetv2_x1_0", pretrained=True).to(device)
validate(model, test_loader)

```

and I got 
1. 74.30 (+0.1) 
2. 74.29 (+0.09).

Can you check your models?

**Comment by yjlee0607** [comment]
1. start on colab
![image](https://user-images.githubusercontent.com/110507135/201587247-62ed770e-4fc4-4063-8922-66ef31f2b55e.png)
2. on my code
![image](https://user-images.githubusercontent.com/110507135/201587710-21c6fc13-160a-4ad4-8d8f-f7b78941671b.png)

**Comment by dbwp031** [comment]
1. start on colab
![image](https://user-images.githubusercontent.com/110507135/201587247-62ed770e-4fc4-4063-8922-66ef31f2b55e.png)
2. on my code
![image](https://user-images.githubusercontent.com/110507135/201587710-21c6fc13-160a-4ad4-8d8f-f7b78941671b.png)

**Comment by chenyaofo** [comment]
Note that different GPU and the version of pytorch would lead to a slight differences in the inference of the model. Thus, it is hard to obtain the same results even with a pretrained model.
In the results produced by your code, the accuracy improves around 0.1%. I think it is quite normal due to the differences mentioned above.

Thanks for your feedback.
